test/fixtures/prepare-execute/nodeflags_only.js

```diff
@@ -0,0 +1,19 @@
+var Liftoff = require('../../..');
+
+var Test = new Liftoff({
+  name: 'test',
+});
+
+Test.on('respawn', function(execArgv) {
+  console.log('saw respawn', execArgv);
+});
+
+Test.prepare({
+  forcedFlags: function(env) {
+    return ['--lazy'];
```

Comment:
This should be modified to `forcedFlags: ['--lazy'],` if `forcedFlags` is not allowed being a function.